metadata.md

```diff
@@ -167,6 +169,73 @@ The sample is a representative volume of wastewater taken from a site which is t
 
 -   **notes**: Any additional notes.
 
+## ContextualMeasurement (ContextualMeasurement.csv) <span id="ContextualMeasurement"><span>
+
+All sorts of measurements that are not performed on the wastewater sample but might provide additional context necessary for the interpretation of the results. 
+
+-   **ID**: (Primary Key) Unique identifier for each contextual measurement.
+
+-   **Site.ID**: (Foreign Key) Links with the Site table to describe the location of measurement.
+
+-   **dateTime**: The date and time the measurement was performed.
+
+-   **type**: The type of measurement that was performed	
+
+    -   `envTemperature`: Environmental temperature.
+    -   `envRainfall`: Amount of precipitation in the form of rain.
+    -   `envSnowFall`: Amount of precipitation in the form of snow.
+    -   `envSnowDepth`: Total depth of snow on the ground.
+    -   `wwFlow`: Flow of wastewater.
+    -   `wwTemp`: Temperature of the wastewater.
+    -   `wwpH`: pH of the wastewater.
+    -   `wwConductivity`: Conductivity of the wastewater.
+    -   `wwTurbidity`: Turbidity of the wastewater.
+    -   `wwTSS`: Total suspended solids concentration of the wastewater.
+    -   `wwCOD`: Chemical oxygen demand of the wastewater.
+    -   `other`: An other type of measurement. Add description to `typeOther`.
+    
+-   **typeOther**: Description of the measurement in case it is not listed in `type`.
+
+-   **typeDescription**: Additional information on the performed measurement.
+
+-   **instrumentName**:	Name of the instrument used to perform the measurement.
```

Comment:
Each `ww..` measure could have its own `typeDescription`, `instrumentName` and `instrumentType`, right?

asking this question because it will be challenging to have the "wide" version of this table because we'd need quite a few measures suces as `wwTemp_insturmentName`. The long version of the table is OK. 

Comment?

Reply:
That's correct. 

I'm not really following with the long and the wide. Why do you need the two options in the first place?

Reply:
Viral studies lab currently all report in the 'long' version. The ERD, SQL and core data dictionary are all organized around a long table -- which is the typical database and data science approach. 

We've tried to accommodate both groups, which is challenging. 
see: https://github.com/Big-Life-Lab/covid-19-wastewater/blob/dev/metadata.md#examples-of-how-to-generate-wide-and-long-variable-and-category-names

It isn't a major issue, I think, but we may want to give examples or try creating long versions of the variable names. There will also be Excel templates and examples. It could be that we recommend the `contextualMeasurement` table be collected in long format. 

We just need to think through any real-world application issues.